packages/web-app-app-store/src/components/AppTags.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="app-tags">
+    <oc-tag
+      v-for="tag in app.tags"
+      :key="`app-tag-${app.id}-${tag}`"
+      size="small"
+      class="oc-text-nowrap"
+      type="button"
+      @click="emitSearchTerm(tag)"
+    >
+      <span class="mark-element">{{ tag }}</span>
+    </oc-tag>
+  </div>
+</template>
+<script lang="ts">
+import { defineComponent, PropType } from 'vue'
+import { App } from '../types'
+
+export default defineComponent({
+  name: 'AppTags',
+  props: {
+    app: {
+      type: Object as PropType<App>,
+      required: true
+    }
+  },
+  emits: ['search'],
```

Comment:
Why does this component makes assumptions and emit a search event, shouldn't it just emit a click event ? 🤔

Reply:
fine by me... care to change it? ;-)